src/style.css

```diff
@@ -1,3 +1,56 @@
-body {
+html {
+  font-family: Roboto, serif;
```

Comment:
Actually, Robot is a "sans-serif" font, so the fallback would be `sans-serif` not `serif`